app/controllers/gigs.js

```diff
@@ -148,28 +148,44 @@ module.exports = function(app) {
       json: true
     }, function(err, response, body) {
       if(err || !body || body.error) {
-        req.flash('error', 'Unable to delete gig');
+        req.flash('error', body.error || 'Unable to delete gig');
       }
       else {
         req.flash('success', 'Gig deleted');
       }
-      return res.redirect('/gigs');
+      return res.redirect('/intranet/gigs');
     });
   });
-  app.put('/gigs/:gig_id', function(req, res) {
+  app.put('/intranet/gigs/:gig_id', function(req, res) {
     request({
-      url: `${SERVICES_URL}/gigs/${req.params.gig_id}`,
-      method: "PUT",
+      url: `${SERVICES_URL}/gigs/claims/${req.params.claim_id}`,
+      method: "GET",
```

Comment:
This is done to validate that the current user is actually the "owner" of the claim. I didn't want to do the token authentication thing again, and we're implying trust between the services, so the wrapping request is used to get the claim requester before issuing the DELETE/PUT action